packages/site-server/src/templates/base.ts

```diff
@@ -23,6 +23,11 @@ export const renderPage = (data: {
       href="https://fonts.googleapis.com/css?family=Material+Icons&display=block"
       rel="stylesheet"
     />
+    <!-- TODO (justinfagnani): only add this in dev. In prod we should have
```

Comment:
Which library needs this?

Reply:
The graphql library that urql depends on.